.github/workflows/mergeability.yml

```diff
@@ -0,0 +1,34 @@
+name: Pull Request Mergeability
+
+on:
+  pull_request_target:
+    branches:
+      - "v3.*"
+
+jobs:
+
+  # IMPORTANT: THIS JOB USES AN UNTRUSTED CHECKOUT!
+  # DO NOT RUN ANY CODE FROM THE REPOSITORY!
+  check_mergeability:
+    name: Check Mergeability
+    if: ${{ github.repository == 'TheThingsNetwork/lorawan-stack' }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Configure Git
+        run: |
+          git config user.name github-actions
```

Comment:
I think this needs to be global,  since you're not in a git repository before the checkout.

https://github.com/TheThingsNetwork/lorawan-stack/runs/4519748526?check_suite_focus=true